docs/Manifest.toml

```diff
@@ -1,8 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.5"
+julia_version = "1.9.0-beta3"
```

Comment:
That should probably stay 1.8.*.
At least that's what we use in the CI, and having a stable version there sounds reasonable to me.